### ISSUE TITLE
Incorrect error message on bun run file with no content

### ISSUE THREAD

**Issue by pinkboid** [oven-sh/bun#6066]
Bun Version: 1.0.2+37edd5a6e389265738e89265bcbdf2999cb81a49
Machine: Darwin 22.3.0 arm64 arm

If you attempt to `bun run` a `.ts` file with no content inside
```
bun init test && cd test && touch index.ts && bun run index.ts
```
it will output a misleading error
```
error: missing script "index.ts"
```
`index.ts` is not missing. It exists. It simply does not have any lines of code to execute. Part of why this is misleading is because it's the same error that will run if you attempt to run a file that actually does not exist in your repository, like `bun run adsljfasdlk.ts`
```
error: missing script "adsljfasdlk.ts"
```

The behavior should be changed to either have no error output on running empty `.ts` files, like `node` does, or it should be a unique error message about how the script has no content rather than the script being missing.

**Comment by Hanaasagi** [comment]
Same to https://github.com/oven-sh/bun/issues/4238, fixed in 1.0.3, please upgrade.

**Comment by pinkboid** [comment]
> Same to #4238, fixed in 1.0.3, please upgrade.
